packages/documentsearch/src/searchview.tsx

```diff
@@ -675,6 +698,7 @@ class SearchOverlay extends React.Component<ISearchOverlayProps> {
               this.props.onHighlightNext();
             }}
             trans={trans}
+            commands={new CommandRegistry()}
```

Comment:
So indeed you are almost there. Now you need to pass `this._commands` from `SearchDocumentView` to here. For that you need to pass it as a new props to `SearchOverlay`. So here the line will become:

```suggestion
            commands={this.props.commands}
```